initializr-generator/src/main/java/io/spring/initializr/generator/version/CompatibilityRange.java

```diff
@@ -34,7 +34,7 @@
  *
  * @author Stephane Nicoll
  */
-public class VersionRange {
+public class CompatibilityRange {
```

Comment:
I don't think we should rename the low-level object type. It does represent a version range and the purpose of this task is to requalify a particular use of it.

Reply:
Agreed upon keeping VersionRange as the underlying name as it does represent a range of the low level object `Version`.

Reply:
FTR, there is actually right now another use of `VersionRange`, see [OnGradleVersionCondition](https://github.com/spring-io/initializr/blob/aaf44d1ec876e8972a9d87897c126db3b8e0d6b8/initializr-generator-spring/src/main/java/io/spring/initializr/generator/spring/build/gradle/OnGradleVersionCondition.java#L39-L43).